sample-apps/blank-java/template-mvn.yml

```diff
@@ -18,3 +18,13 @@ Resources:
         - AWSXrayWriteOnlyAccess
         - AWSLambdaVPCAccessExecutionRole
       Tracing: Active
+      Layers:
```

Comment:
I wouldn't make examples of Lambda functions using Layers for application dependencies

Reply:
Why not?

Reply:
It'd be good to provide one example of this, and I think blank-java is a good candidate for it. Some of the other languages in this repo show this pattern as well (Node, Python).

Reply:
If you provide an example of this, it absolutely should be in a separate example. This is not what I would expect from 'blank'. I would call it 'app-dependency-layer' or something similar.

Reply:
Blank does not mean "empty". You can get an empty function from the management console. It's a sample for demonstrating and testing service features. Layers is an important feature, no?